test/unit/state/frameStateUtilTest.js

```diff
@@ -17,92 +17,6 @@ describe('frameStateUtil', function () {
     this.windowState = Immutable.fromJS(Object.assign({}, defaultWindowStore.toJS()))
   })
 
-  describe('cloneFrame', function () {
-    before(function () {
-      this.frameOpts = {
-        key: 1,
-        tabId: 2,
-        parentFrameKey: 3,
-        audioMuted: true,
```

Comment:
I think this is the only one we need to handle and we can do that in electron -  https://github.com/brave/electron/issues/126